src/templates/results/components/PredictedLegend.js

```diff
@@ -0,0 +1,75 @@
+import { Row, Col } from 'react-bootstrap';
+
+
+const PredictedLegend = (props) => {
```

Comment:
Seems like something that should be generated by the library. Why is it necessary to build the legend yourself?

Reply:
Ok next files explains it, might be nicer if we could get something out of the box for this but not necessary, and not for this PR.

Reply:
Was thinking about it at first, could probably do it, also like to practice with these kind of things. Could do it if there is time in the future

Reply:
:+1: